stories/tabs.stories.js

```diff
@@ -13,6 +13,11 @@ const titleAsComponent = (
   </>
 );
 
+const alertTabOnclick = activateTab => {
+  alert('tab clicked!');
+  activateTab();
```

Comment:
here's how to actually use it. Just call the callback passed as the arg if you want to have it do its default tab activation behavior. this is useful e.g. for tracking